spec/unit/policy_builder/policyfile_spec.rb

```diff
@@ -150,7 +150,7 @@ def initialize_pb
       expect(initialize_pb.temporary_policy?).to be_falsey
     end
 
-    context "chef-solo" do
+    context "#{Chef::Dist::SOLOEXEC}" do
```

Comment:
this probably shouldn't be done since i don't think there's any technical or legal reason to do this -- but it breaks grepping through the file for strings in the rspec output, so there's a developer-workflow reason to keep it with the old string.  if it causes confusion or something later we can revisit this, but i'd rather not do it right now.